wiki/Client/Beatmap_editor/Timing/en.md

```diff
@@ -1,25 +1,96 @@
+  - copy timing
+  - paste timing
+---
 
-*See also: [How to time songs](/wiki/Guides/How_to_Time_Songs)*
+# Timing
 
-![Screenshot of the timing tab in the editor](/wiki/shared/timing/Timing_base.jpg)
-
-The **timing tab** of the [beatmap editor](/wiki/Client/Beatmap_editor) lets mappers change the timing settings of a [beatmap](/wiki/Beatmap). Timing is usually the first step in creating a new beatmap.
+*For a timing setup guide, see: [How to time songs](/wiki/Guides/How_to_Time_Songs)*\
+*See also: [Beatmapping/Timing](/wiki/Beatmapping/Timing)*
 
-Timing consists of a song's [offset](/wiki/Offset), [beats per minute (BPM)](/wiki/Music_theory/Tempo), and [meter](https://en.wikipedia.org/wiki/Metre_(music)). Correct timing on beatmaps is necessary for players to maintain [accuracy](/wiki/Gameplay/Accuracy) while playing. Incorrectly timed beatmaps cannot pass through the [beatmap ranking procedure](/wiki/Beatmap_ranking_procedure).
+**Timing** is the [beatmap editor](/wiki/Client/Beatmap_editor)'s tab for changing and setting up timing of a [beatmap](/wiki/Beatmap), which is essential for correct song representation. It contains related settings and helpers, and also a [dedicated window](#timing-setup-panel) for working with multiple [timing points](#timing-points), both for musical structure description and beatmap design purposes.
 
-## Timing and Control Points window
+## Timing points
 
-![Screenshot of the Timing and Control Points window](/wiki/shared/timing/TimingSetup.png)
+*See also: [Offset](/wiki/Offset)*
 
-Timing points can be added from the `Timing and Control Points` window in the editor (default shortcut `F6`). Timing points divide a song into timing sections to accurately reflect changes to a song's timing.
+In [mapping](/wiki/Beatmapping), a *timing point*, colloquially called an *offset*, is a way to apply common settings, such as [timing](/wiki/Beatmapping/Timing), [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) multiplier, or [hitsounds](/wiki/Beatmapping/Hitsound)' volume, to a whole section of the beatmap. In osu!, there are two types of timing points.
 
 ### Uninherited timing point
 
-An uninherited timing point, displayed as a red line, is the base for a timing section's BPM, offset, and meter. Every beatmap must have at least one red line, however some songs require multiple due to more complex composition or timing inaccuracies caused by live performance.
+::: Infobox
+![](img/uninherited-offsets.png "Several uninherited timing points located in the timing setup panel")
+:::
+
+An **uninherited** timing point has its own timing settings. Several such points are used to describe timing changes in the song, such as [tempo](/wiki/Music_theory/Tempo) or beat irregularities, or different size of the [time signature](/wiki/Music_theory/Time_signature). Adding a new uninherited timing point resets the metronome, and makes it count beats from the timing point's offset, specified in milliseconds.
```

Comment:
```suggestion
An **uninherited** timing point has its own timing settings. Several of such points are used to represent timing changes in the song, such as [tempo](/wiki/Music_theory/Tempo), beat irregularities, omitting a barline in [osu!taiko](/wiki/Game_mode/osu!taiko) or [osu!mania](/wiki/Game_mode/osu!mania), or a different size of [time signature](/wiki/Music_theory/Time_signature). Adding a new uninherited timing point resets the metronome, making it count beats from the timing point's offset, specified in milliseconds.
```

Reply:
fwiw we always treat bar line related things as "timing", even tho it might not make a lot of sense at first glance